lightning/src/onion_message/messenger.rs

```diff
@@ -735,10 +735,13 @@ where
 		&self, contents: T, destination: Destination, reply_path: Option<BlindedPath>,
 		log_suffix: fmt::Arguments
 	) -> Result<SendSuccess, SendError> {
-		let intro_node_logger =
-			WithContext::from(&self.logger, Some(destination.first_node()), None);
+		let mut log_wrapper = WithContext::from(&self.logger, None, None);
```

Comment:
I guess we can just call this `logger` now and use it everywhere for consistency / simplicity.